src/main.py

```diff
@@ -45,44 +43,59 @@ def draw_window(player):
     WIN.blit(frog, (player.x, player.y))
     pygame.display.update()
 
+
 def log_game():
     """Initializes console for logging messages"""
     logging.basicConfig(level=logging.INFO)
     logging.info("Welcome to The Froggerithm!")
 
+
+def move_player(player, keys_depressed):
+    """Handles player movement"""
+    x_change = 0
+    y_change = 0
+
+    if keys_depressed[pygame.K_LEFT] or keys_depressed[pygame.K_a]:  # Left arrow key or a
+        x_change -= MOVEMENT_DISTANCE_X
+        y_change = 0
```

Comment:
Is there any need to reassign `y_change` or `x_change` if it's already set to 0 at the start of the function? It's a negligible performance difference, but when we're running 100+ copies of the game in a training generation, it'll add up.

Reply:
I think the nest positions will be fine because the logs move continuously. I guess there's no need to reassign those variables when we don't need to, I just thought that doing it that way might help avoid any diagonal movement but I could be wrong.

Reply:
Ah yeah good point about the nests. Should be fine then. For the re-assignment, go ahead and take those out I'd say. If it's a problem then let me know. Otherwise I'd rather not have it in there.